torchvision/datasets/folder.py

```diff
@@ -38,15 +38,37 @@ def make_dataset(
     extensions: Optional[Tuple[str, ...]] = None,
     is_valid_file: Optional[Callable[[str], bool]] = None,
 ) -> List[Tuple[str, int]]:
+    """Generates a list of samples of a form (path_to_sample, class).
+
+    Args:
+        directory (str): root dataset directory
+        class_to_idx (Dict[str, int]): dictionary mapping class name to class index
+        extensions (Optional[Tuple[str, ...]], optional): A list of allowed extensions.
+            Both extensions and is_valid_file should not be passed. Defaults to None.
```

Comment:
This docstring and the one of `DatasetFolder` actually do not say that if both None it is also a problem.

Reply:
Should be addressed?